### ISSUE TITLE
Work with startOf?

### ISSUE THREAD

**Issue by aljones** [WhoopInc/frozen-moment#13]
Doesn't seem to work with startOf. 

```
> var moment = require('frozen-moment')
> var now = moment()
> var frozen = moment.frozen()
> now.format()
'2015-11-05T15:25:24-05:00'
> frozen.format()
'2015-11-05T15:25:48-05:00'
> now.startOf('year').format()
'2015-01-01T00:00:00-05:00'
> frozen.startOf('year').format()
'2015-11-05T15:25:48-05:00'
```

**Comment by butterflyhug** [comment]
Thanks for the bug report!  I'll release a fix momentarily.

**Comment by butterflyhug** [comment]
Should be fixed in 0.3.0.  Please open a new issue if you're still having problems (with this, or any other aspect of frozen-moment).
